nixos/modules/services/monitoring/tuptime.nix

```diff
@@ -32,7 +32,7 @@ in {
 
     environment.systemPackages = [ pkgs.tuptime ];
 
-    users.users.tuptime.description = "tuptime database owner";
+    users.users._tuptime.description = "tuptime database owner";
```

Comment:
Why the underscore?

Reply:
upstream changed to this, i don't know why but thought matching that change was best

Reply:
agreed.